raw_data/preprocessed/africa/guinea-bissau_portuguese.txt

```diff
@@ -0,0 +1,199 @@
+
+Objectivos e Medidas
+(6) A existência de um ambiente propício à saúde e bem-estar das pessoas e ao desenvolvimento social e cultural das comunidades, bem como a melhoria da qualidade de vida pressupõe a adopção de medidas que visem, designadamente: [a] o desenvolvimento económico, social e durável; [b] a manutenção e o equilíbrio entre a satisfação das necessidades básicas das pessoas e a capacidade de resposta da natureza; [c] a prestação da maior atenção à qualidade do ambiente urbano através de uma eficaz aplicação de política do poder local; [d] a manutenção dos ecossistemas terrestres, marinhos e ecossistema de transição; [e] a conservação da natureza; [f] a protecção dos habitats; [g] a delimitação dos níveis de qualidade dos componentes ambientais; [h] a definição de uma política energética baseada no aproveitamento das fontes limpas ou energias renováveis de forma racional e durável dos recursos naturais, diversificação e descentralização das fontes de produção na racionalização do consumo; [i] a promoção da participação das populações nos processos de tomada de decisão; [j] a educação e formação ambiental; [k] a sustentabilidade da floresta; [l] a articulação com países limítrofes de acções de defesa ambiental e de aumento da qualidade de vida das populações fronteiriças; [m] a prevenção da erosão do solo, interior e costeira; [n] a promoção de acordo com outros sectores da vida nacional, da defesa do consumidor; [o] a promoção da agricultura produtiva e durável que contribua para vitalidade social e económica das zonas rurais e que garante um desenvolvimento equilibrado das zonas rurais e urbanas; [p] a revitalização da fauna e da flora; [q] a protecção do ar e do clima; [r] a adequada gestão dos resíduos; [s] o reforço das acções de defesa e recuperação do património natural construído e recuperação de áreas degradadas; [t] a garantia do mínimo “impacte” ambiental das actividades e utilização da melhor tecnologia disponível na minimização dos” impactes” ambientais; [u] promoção e valorização dos saberes tradicionais na gestão dos recursos naturais; [v] disseminação de cultura e gestão ambiental através dos três ‘'R'', ou seja, Reduzir, Reciclar e Reutilizar.
```

Comment:
The quotations around ‘'R'' are different from the ordinary quotations used ('' instead of ").

Reply:
I will replace all of the problematic quotation marks in this legislation. Thanks for pointing them out.